lib/samples/control_annotation_sublayer_visibility/README.md

```diff
@@ -0,0 +1,59 @@
+# Control annotation sublayer visibility
+
+Use annotation sublayers to gain finer control of annotation layer subtypes.
+
+![Image of control annotation sublayer visibility](control_annotation_sublayer_visibility.png)
+
+## Use case
+
+Annotation, which differs from labels by having a fixed place and size, is typically only relevant
+at particular scales. Annotation sublayers allow for finer control of annotation by allowing
+properties (like visibility in the map and legend) to be set and others to be read (like name) on
+subtypes of an annotation layer.
+
+An annotation dataset which marks valves as "Opened" or "Closed", might be set to display the "
+Closed" valves over a broader range of scales than the "Opened" valves, if the "Closed" data is
+considered more relevant by the map's author. Regardless, the user can be given a manual option to
+set visibility of annotation sublayers on and off, if required.
+
+## How to use the sample
+
+Start the sample and take note of the visibility of the annotation. Zoom in and out to see the
+annotation turn on and off based on scale ranges set on the data.
+
+Use the checkboxes to manually set "Open" and "Closed" annotation sublayers visibility to on or off.
+
+## How it works
+
+1. Create a new `ArcGISMapView` with an `ArcGISMapViewController`.
```

Comment:
```suggestion
```
Since this step isn't in the design readme I don't think we need to include it (our comments in the code is sufficient). Note you'll need to amend the numbering accordingly.